packages/atomic/src/components/common/button.spec.ts

```diff
@@ -1,70 +1,60 @@
 import {createRipple} from '@/src/utils/ripple';
-import {fireEvent, within} from '@storybook/test';
-import {html, nothing, render} from 'lit';
-import {vi} from 'vitest';
-import {button, ButtonProps} from './button';
+import {fixture} from '@/vitest-utils/testing-helpers/fixture.js';
+import {userEvent} from '@vitest/browser/context';
+import {html, nothing} from 'lit';
+import {vi, expect, describe, it} from 'vitest';
+import {button, ButtonProps} from './button.js';
 
-vi.mock('@/src/utils/ripple', () => ({
-  createRipple: vi.fn(),
-}));
+vi.mock('@/src/utils/ripple', {spy: true});
```

Comment:
In my personal opinion, I prefer having one-liner import-like mock-ups to identify what module is mocked and leaving the implementation and whatnot to the guts of the tests.

My rationale is that when I need to play with the mock, it's in the tests, and having to scroll back up, check, go back down is tedious.